### ISSUE TITLE
Integrate with Kiosker iOS app

### ISSUE THREAD

**Issue by John5373** [thomasloven/hass-browser_mod#248]
Could you add a service to integrate with the Kiosker iOS app? Specifically, I am trying to enable/disable the screensaver through the app's javascript functionality:

[](https://docs.kiosker.io/#/javascript?id=screensaver)

Thanks!

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.

**Comment by xiaohuim** [comment]
+1 for this! 
This feature is really heplful for people who use iPad as a dashboard. Can you please add support for Kiosker? 
Kiosker.io has all the javascript related documents [https://docs.kiosker.io/#/javascript](https://docs.kiosker.io/#/javascript)
Big thanks!

**Comment by xiaohuim** [comment]
@maniscalcojohn  I figured out a easy (or temporarily) way to do it with Kiosker. So I have an iPad as dashboard with the screen on 24/7, and I want the screen to light up (like 60% the full brightness) when motion detected and brightness to 0 when no one is around. This is how I did it:
<img width="1052" alt="截屏2023-09-22 21 38 38" src="https://github.com/thomasloven/hass-browser_mod/assets/3395293/e9b6b953-4328-4bdf-8b3d-0901be1feba0">

Kiosk website has this function on their doc:
```
function kioskerIntegration(data) {
    const string = JSON.stringify(data)
    try {
      window.webkit.messageHandlers.callback.postMessage(string);
    } catch(err) {
      console.log("Can't reach Kiosker");
    }
  }
```
Basically what we need do is just stringify the JSON data and pass that to the `window.webkit.messageHandlers.callback.postMessage()` function.

Hope this can help.
